subworkflows/nf-core/fasta_gxf_busco_plot/main.nf

```diff
@@ -0,0 +1,168 @@
+                                                //
+    val_lineages                                // [ val(lineage) ]
+    val_busco_lineages_path                     // val(path); Optional; Set to [] if not needed
+    val_busco_config                            // val(path); Optional; Set to [] if not needed
+
+    main:
+    ch_versions                                 = Channel.empty()
+    ch_db_path                                  = val_busco_lineages_path
+                                                ? Channel.of(file(val_busco_lineages_path, checkIfExists: true))
+                                                : Channel.of(null)
+    ch_config_path                              = val_busco_config
+                                                ? Channel.of(file(val_busco_config, checkIfExists: true))
+                                                : Channel.of(null)
+
+    // MODULE: BUSCO_BUSCO as BUSCO_ASSEMBLY
+    ch_busco_assembly_inputs                    = ch_fasta
+                                                | combine(
+                                                    Channel.of(val_mode)
+                                                )
+                                                | combine(
+                                                    Channel.fromList(val_lineages)
+                                                )
+                                                | map { meta, fasta, mode, lineage ->
+                                                    [
+                                                        meta + [ mode:mode, lineage:lineage ],
+                                                        fasta, mode, lineage
+                                                    ]
+                                                }
+                                                | combine(
+                                                    ch_db_path
+                                                )
+                                                | combine(
+                                                    ch_config_path
+                                                )
+
+    BUSCO_ASSEMBLY(
+        ch_busco_assembly_inputs.map { meta, fasta, mode, lineage, db, config -> [ meta, fasta ] },
+        ch_busco_assembly_inputs.map { meta, fasta, mode, lineage, db, config -> mode },
+        ch_busco_assembly_inputs.map { meta, fasta, mode, lineage, db, config -> lineage },
+        ch_busco_assembly_inputs.map { meta, fasta, mode, lineage, db, config -> db ?: [] },
+        ch_busco_assembly_inputs.map { meta, fasta, mode, lineage, db, config -> config ?: [] }
+    )
+
+    ch_assembly_batch_summary                   = BUSCO_ASSEMBLY.out.batch_summary
+    ch_assembly_short_summaries_txt             = BUSCO_ASSEMBLY.out.short_summaries_txt
+    ch_assembly_short_summaries_json            = BUSCO_ASSEMBLY.out.short_summaries_json
```

Comment:
```suggestion
    ch_assembly_short_summaries_json            = BUSCO_ASSEMBLY.out.short_summaries_json
    ch_assembly_full_table                      = BUSCO_ASSEMBLY.out.full_table
```
Thanks for adding this as an subworkflow @GallVp.

Our [genomeqc](https://github.com/nf-core/genomeqc) pipeline does some plotting based on the `full_table.tsv` output from BUSCO.

Would it be OK if your subworkflow emits the associated channel so that we can access this file (suggested changes above and below)?